### PR TITLE
[skip ci] Initial implementation of VIC SSO integration test

### LINKED ISSUE #5689: VIC 1.2 SSO integration test
User statement:  As a customer of VIC 1.2, I want to be assured through testing that the new SSO feature is working as expected

Details:
We will need to integrate testing between all the components of VIC - admiral/harbor/engine and we will need to at a minimum test this in a standard VC topo (integrated PSC) and an ELM topo (discrete PSC)

Acceptance criteria:
SSO integration test is well documented in a test plan and executed manually or automated.

### PR DESCRIPTION
**Pull request by mhagen-vmware**
fixes #5689

### CODE REVIEW

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+4. Navigate to the admiral, harbor, demo VCH pages
+5. Login into the vCenter using correct credentials
+6. Navigate to the admiral, harbor, demo VCH pages
+7. Wait 10 minutes
+8. Navigate to the admiral, harbor, demo VCH pages
+9. Wait another 50 minutes (60 minutes total)
+10. Navigate to the admiral, harbor, demo VCH pages
+
+# Expected Outcome:
+* Steps 1,2,4-8 should all succeed without error
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: Should this be `5-8` instead of `4-8`? Step 4 should fail upon entering incorrect creds - Harbor and Admiral are only started after correct creds have been entered.

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* Step 3 should result in error and prompt the user for the correct credentials again
+* Step 4 should fail and the user should be prompted for login credentials at every page
```
Comment (anchal-agrawal) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: Currently, upon entering incorrect creds, Harbor and Admiral aren't started - so their pages won't load.

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# References:
+[1- VIC SSO Feature](TBD - waiting on official docs to link)
+
+# Environment:
+This test requires that a vCenter server with integrated PSC and with a discrete PSC is running and available
+
+# Test Steps:
+1. Download and install the VIC OVA from here:
+[OVA Releases](https://console.cloud.google.com/storage/browser/vic-product-ova-releases/?project=eminent-nation-87317&authuser=1)
+2. Navigate to the file server page https://<VIC>:9443
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: I'm calling this the `Getting Started Page` now

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+6. Navigate to the admiral, harbor, demo VCH pages
+7. Wait 10 minutes
+8. Navigate to the admiral, harbor, demo VCH pages
+9. Wait another 50 minutes (60 minutes total)
+10. Navigate to the admiral, harbor, demo VCH pages
+
+# Expected Outcome:
+* Steps 1,2,5-8 should all succeed without error
+* Step 3 should result in error and prompt the user for the correct credentials again
+* Step 4 should fail and the user should be redirected to the fileserver page in order to login correctly
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: This feature is not implemented

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+* Step 6 and 8 should succeed and the user should not be prompted for login credentials to any of the pages
+* Step 10 should fail and the user should be prompted for login credentials again
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: This shouldn't happen because the token is valid for like a month and is set to auto refresh

Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: I think all steps after 6 are unnecessary

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: I thought @anchal-agrawal was working on implementing a 1hr timeout.. we can't leave that session going for that long can we?

Comment (anchal-agrawal) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: We refresh the tokens every hour to ensure we have valid tokens for engine, harbor and admiral since the token expiry period can be configured by users.

Comment (mhagen-vmware) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: @anchal-agrawal So should this test work as written?

File: tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md (hunk 4 above)
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group19-VIC-Product/19-8-VIC-SSO.md]: np, I still want this test as written since that is the ultimate desired behavior.
